drf_braces/fields/modified.py

```diff
@@ -24,6 +24,24 @@ def quantize(self, value):
         return super(DecimalField, self).quantize(value)
 
 
+class CurrencyField(DecimalField):
+    """
+    Currency field subclass of Decimal used for rounding currencies
+    to two decimal places.
+    """
+    precision = 2
```

Comment:
should not as long as `self.validate_precision` is changed to not enforce that validation